### PR TITLE
Added support for 'instance_types_filters'.

### PR DESCRIPTION
**Pull request by sharadkesarwani**
Added support for 'instance_types_filters'.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,6 +39,31 @@ module "ocean-aws-k8s-vng_gpu" {
   #instance_types = ["g4dn.xlarge","g4dn.2xlarge"] # Limit VNG to specific instance types
   spot_percentage = 50 # Change the spot %
 }
+
+## Create Ocean Virtual Node Group (launchSpec) with instance_type_filters ##
+module "ocean-aws-k8s-vng" {
+  source  = "spotinst/ocean-aws-k8s-vng/spotinst"
+  ocean_id = "o-123456"
```
Comment (chandra1-n) [inline README.md:46]: use reference - module.ocean-aws-k8s.ocean_id for ocean_id

File: examples/basic/main.tf [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+terraform {
+  required_providers {
+    spotinst = {
+      source = "spotinst/spotinst"
+    }
+  }
+}
+## Create Ocean VNG on Spot.io ##
+module "ocean-aws-k8s-vng" {
+  source                                          = "spotinst/ocean-aws-k8s-vng/spotinst"
```
Comment (chandra1-n) [inline examples/basic/main.tf:14]: add other required parameters as well

**Review by chandra1-n — CHANGES_REQUESTED** [review]
please fix the comments

File: README.md (hunk 1 above)
Comment (sharadkesarwani) [reply README.md:46]: Reference added.

**Review by chandra1-n — APPROVED** [review]

**Pull request merged**